server/game/cards/02_SHD/units/CoruscantDissident.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { TargetMode, WildcardCardType } from '../../../core/Constants';
+
+export default class CoruscantDissident extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9115773123',
+            internalName: 'coruscant-dissident'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'On attack: You may ready a resource',
```

Comment:
on title you can omit "may" part as he will have the pass ability button and "on attack"

Reply:
I have updated this to say 'Ready a resource'. Please let me know if you want that worded differently and if there are any other issues with the implementation code